apidoc/Titanium/Media/Media.yml

```diff
@@ -2104,8 +2106,15 @@ properties:
     description: |
         The allowMultiple property is only available on Android API 18 and above.
     type: Boolean
-    platforms: [android]
-    since: "6.0.0"
+    platforms: [android, iphone, ipad]
+    osver: {ios: {min: "14.0"}}
+    since: { android: "6.0.0", iphone: "9.2.0", ipad: "9.2.0" }
+  - name: selectionLimit
+    summary: Specifies number of assets that can be selected.
+    type: Boolean
+    platforms: [iphone, ipad]
+    osver: {ios: {min: "14.0"}}
+    since: "9.2.0"
```

Comment:
I think "selectionLimit" should be ignored unless "allowMultiple" is set `true`. Especially since Android does not support this property (not even natively). Requiring the app developer to set "allowMultiple" to `true` would then make the code portable.

We should also document that setting this to zero allows you to select an unlimited number of media.

Reply:
Yes. Updated "selectionLimit" to work when "allowMultiple" is `true` only.